fjord/translations/models.py

```diff
@@ -510,9 +434,13 @@ def translate(self, instance, src_lang, src_field, dst_lang, dst_field):
                 metadata=metadata)
             return
 
-        # If the src -> dst isn't a supported pair, log an issue for
-        # metrics purposes and move on.
-        if (lc_src, dst_lang) not in gengo_api.get_language_pairs():
+        # If this is 'standard' tier and if the src -> dst isn't a
+        # supported pair, log an issue for metrics purposes and move
+        # on. We only do this for 'standard' tier since the 'machine'
+        # tier has a different set of pairs which aren't available to
+        # us.
+        if ((self.gengo_tier == 'standard'
```

Comment:
I talked about this in my other comment. The gist of it is that testing this is entirely manual and difficult and I'm doing it on a live server so it has monetary consequences. This was the best I could come up with that didn't involve lots of code changes from what I had before, didn't require much extra testing and centralized the code for easier maintenance going forward.

Generally, this code doesn't change over time. We haven't added new features or adjusted existing behavior. The only time this code changes is when Gengo changes some aspect of their API. I'm pretty sure every time they do that, we have to make substantive changes to our code--it's not usually a quick fix.

Having said that, I could do another "flag" thing like I did with the "gengo_watch_balance" flag rather than doing a tier check. I'll look at doing that today.